bedrock/mozorg/templates/mozorg/home/includes/ctd-promo-de.html

```diff
@@ -0,0 +1,61 @@
+{#
+  This Source Code Form is subject to the terms of the Mozilla Public
+  License, v. 2.0. If a copy of the MPL was not distributed with this
+  file, You can obtain one at https://mozilla.org/MPL/2.0/.
+#}
+
+{% block content %}
+  <section class="ctd-promo">
+
+    <!-- CTD Promo for non-Firefox users -->
+    <div class="ctd-promo-non-fx">
+      {% call split(
+        block_class='mzp-t-split-nospace mzp-l-split-center-on-sm-md t-pocket-promo',
+        theme_class='mzp-c-split-bg',
+        body_class=None,
+        media_class='mzp-l-split-h-center mzp-l-split-v-center',
+        media_after=False,
+        image=resp_img(
+            url='img/home/2023/ctd-onboarding.png',)
+        ) %}
+    <div class="mzp-c-wordmark mzp-t-wordmark-md mzp-t-product-family">
+      Firefox
+    </div>
+    <h2>Dein Browser ist soooooo Standard</h2>
+    <p>Wir sagen nicht, dass Firefox auf jeden Fall der bessere Browser für dich ist. Wir sagen nur, dass es sich lohnen könnte, den Browser zu wechseln.
+    </p>
+    <small>(Wir haben auch ein paar wirklich gute Argumente.)</small>
```

Comment:
This should probably be a in paragraph. Then you won't need to style this as `display:block`.

Reply:
I figured I'd just use `small` since if I used a `p` tag, i'd still need to give it a specific class + reduce default font size, so opted to use this instead.